templates/docker/docker-compose.production.example.yml

```diff
@@ -0,0 +1,19 @@
+# Production Docker Setup Node
+# Redis
+# redis:
+#   image: redis
+
+# Postgres
+# db:
+#   image: postgres
+
+web:
+  build: .
+  command: npm start
+  # environment:
+  #   REDIS_HOST: redis
+  # links:
+  #   - redis
+  #   - db
+  ports:
+    - "80:3000"
```

Comment:
I'll be honest.  Docker compose is much easier and it's built in and supported by docker.  This also lets people run production containers on their own computer without having to set up yet another vm.